app/controllers/manager/statistics/employees_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Manager
+  module Statistics
+    class EmployeesController < ApplicationController
+      layout false
+
+      def index; end
+
+      def academic
+        @series = Employee.active.academic.group('titles.name').count
```

Comment:
```suggestion
        @series ||= Employee.active.academic.group('titles.name').count
```

Reply:
`@series` burada bir kez kullanıldığı için bu yöntem ile diğeri arasında bir fark oluşmuyor yani bu yöntem ile cachelemek bize fayda sağlamıyor.